keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py

```diff
@@ -54,14 +54,14 @@ def __init__(self, r_distance, theta_width, phi_width, drop_rate=None, **kwargs)
         super().__init__(**kwargs)
 
         if r_distance < 0:
-            raise ValueError("r_distance must be >=0.")
+            raise ValueError(f"r_distance must be >=0, but got {r_distance}")
         if theta_width < 0:
-            raise ValueError("theta_width must be >=0.")
+            raise ValueError(f"theta_width must be >=0, but got {theta_width}")
         if phi_width < 0:
-            raise ValueError("phi_width must be >=0.")
+            raise ValueError(f"phi_width must be >=0, but got {phi_width}")
         drop_rate = drop_rate if drop_rate else 0.0
         if drop_rate > 1:
-            raise ValueError("drop_rate must be <=1.")
+            raise ValueError(f"drop_rate must be <=1, but got {drop_rate}")
```

Comment:
can drop_rate be negative?  I think it must be 0 < x < 1, right?

Reply:
I think you're right, it should not be negative. I'm no expert at this but I guess as it represents the fraction of units to drop, it should be between 0 and 1. 
Besides the `Dropout` layer of `Keras` also needs a rate between 0 and 1, so.